src/popup-new.html

```diff
@@ -93,6 +94,14 @@ <h4 class="i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></h4>
               <span class="fx-relay-masks-limit-upgrade-string"></span>
             </div>
 
+            <!-- Premium Masks / Error / Cannot Make Mask -->
+            <div tabindex="0" class="fx-relay-masks-error-message">
```

Comment:
@Vinnl I was making this keyboard friendly, but realized we need some level of ARIA support too. Flagging this now (no action needed) to surface for the _big one™_ review later.

Reply:
Yep, manual tabindexes always give me pause - sounds like the close icon should be inside a button, perhaps? And possibly a [`role="alert"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role)?

Reply:
Added in 9428bd79504cd76a16256321e0d352348731ef2e